WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Shipping Address Validation/ShippingLabelAddressFormViewController.swift

```diff
@@ -94,7 +107,7 @@ private extension ShippingLabelAddressFormViewController {
     }
 
     func updateTopBannerView() {
-        topBannerView.isHidden = viewModel.addressValidationError?.generalError != nil
+        topBannerView.isHidden = viewModel.addressValidationError?.generalError == nil
```

Comment:
I think we can turn this into
```swift
topBannerView.isHidden = !viewModel.shouldShowTopBannerView
```
or
```swift
topBannerView.isHidden = !viewModel.hasAddresValidationError()
```

What do you think?

Reply:
It makes sense to me. I applied the changes here e5c58a2ba3646b367d625f1a133da831bc30299e